aws/aws.go

```diff
@@ -478,6 +487,44 @@ func GetAllResources(targetRegions []string, excludeAfter time.Time, resourceTyp
 		count++
 	}
 
+	// Global Resources - These resources are global and do not belong to a specific region
+
+	logging.Logger.Infof("Checking region [%d/%d]: %s", count, totalRegions, GlobalRegion)
+
+	// As there is no actual region named global we have to pick a random one just to create the session
+	randomRegion, err := getRandomRegion()
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	session, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(randomRegion)},
+	)
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	globalResources := AwsRegionResource{}
+
+	// IAM Users
+	iamUsers := IAMUsers{}
+	if IsNukeable(iamUsers.ResourceName(), resourceTypes) {
+		userNames, err := getAllIamUsers(session, configObj)
+
+		if err != nil {
+			return nil, errors.WithStackTrace(err)
+		}
+		if len(userNames) > 0 {
+			iamUsers.UserNames = awsgo.StringValueSlice(userNames)
+			globalResources.Resources = append(globalResources.Resources, iamUsers)
+		}
+	}
+	// End IAM Users
```

Comment:
Somewhere in here we need a check to make sure the global region is in `targetRegions`. This is to handle the case where someone puts `global` in the excluded regions list.